hidio/examples/hierarchical_goaltask.gin

```diff
@@ -141,4 +141,5 @@ TrainerConfig.summarize_grads_and_vars=0
 TrainerConfig.summary_interval=20
 TrainerConfig.use_rollout_state=True
 TrainerConfig.replay_buffer_length=%replay_buffer_length
-TrainerConfig.priority_replay_alpha=1.0
\ No newline at end of file
+TrainerConfig.priority_replay_alpha=1.0
+TrainerConfig.epsilon_greedy=0.
```

Comment:
Are you sure it's not because epsilon_greedy=0 makes this change better?

Reply:
Yeah I tested with epsilon_greedy=0. and the default 0.1 on multiple seeds with the previous alf commit. While some seeds are just as good as the paper, many end up being a bit worse.